partner_contact_lang/views/res_partner_view.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Copyright 2020 Pesol <pedro.gonzalez@pesol.es>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">view.partner.address.form.extension</field>
+        <field name="model">res.partner</field>
+        <field name="priority">100</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='child_ids']/form/sheet/field[@name='lang']"
+                position="replace"
+            >
+                <field name="active_lang_count" invisible="1" />
+                <group attrs="{'invisible': [('active_lang_count', '&lt;=', 1)]}">
```

Comment:
Sorry for late answe: Looks great, maybe you could add  the col and colspan attributes to the group to have the exact same col width:

```xml
        <field name="arch" type="xml">
            <xpath
                expr="//field[@name='child_ids']/form/sheet/field[@name='lang']"
                position="replace"
            >
                <field name="active_lang_count" invisible="1" />
                <group col="12" attrs="{'invisible': [('active_lang_count', '&lt;=', 1)]}">
                    <group colspan="5">
                        <field name="lang" />
                    </group>
                </group>
            </xpath>
        </field>
```
These column definitions are defined in res_partner form view ([here](https://github.com/odoo/odoo/blob/30be09cdb7d5b80fb1e0221f58024586bd72ac61/odoo/addons/base/views/res_partner_views.xml#L357))


But so far: Great work, thank you for contrib, thumbs up from me :+1: